nautobot/docs/models/dcim/locationtype.md

```diff
@@ -0,0 +1,27 @@
+# Location Types
+
+<!-- markdownlint-disable MD036 -->
+_Added in version 1.4.0_
+<!-- markdownlint-enable MD036 -->
+
+Before defining individual Locations, you must first define the hierarchy of Location Types that you wish to use for the organization of your network. An example hierarchy might be `Building ← Floor ← Room`, but you might have more or fewer distinct types depending on your specific organizational requirements.
```

Comment:
I've gone back and forth on this one. I was thinking "a Floor belongs to a Building" rather than the other way around, plus the data model has the foreign key from child to parent, not parent to child. But I'm open to discussion on this.